From 1dc4c484520fd34f0db971bd7158dc60ce6e7a50 Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Sat, 29 Jun 2019 16:22:36 +0800
Subject: [PATCH] always write fstab after install

We should write the storage after doing the installation
which always writes fstab after instal, make sure the generated
/etc/fstab not overwritten by the one from package install.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 pyanaconda/payload/dnfpayload.py | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/pyanaconda/payload/dnfpayload.py b/pyanaconda/payload/dnfpayload.py
index 512f217..cd6b209 100644
--- a/pyanaconda/payload/dnfpayload.py
+++ b/pyanaconda/payload/dnfpayload.py
@@ -1485,11 +1485,6 @@ class DNFPayload(payload.PackagePayload):
             if ks_repo.excludepkgs:
                 f.write("exclude=%s\n" % ",".join(ks_repo.excludepkgs))
 
-    @property
-    def needs_storage_configuration(self):
-        """Should we write the storage before doing the installation?"""
-        return True
-
     def post_setup(self):
         """Perform post-setup tasks.
 
-- 
2.7.4

